docs/.vuepress/theme/components/AposFeedback.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <!-- Based on https://github.com/ipfs/ipfs-docs-v2/blob/master/docs/.vuepress/theme/components/Feedback.vue -->
+  <div class="c-feedback">
+    <h3>{{ titleTxt }}</h3>
+    <div v-if="!voteSubmitted" class="feedback-actions">
+      <button :title="yesTxt" v-on:click="sendFeedback(evtYes)">
+        {{ yesTxt }}
+      </button>
+      <button :title="noTxt" v-on:click="sendFeedback(evtNo)">
+        {{ noTxt }}
+      </button>
+    </div>
+    <div v-if="voteSubmitted" class="feedback-result">
+      <p>{{ thanksTxt }}</p>
+    </div>
+    <div v-if="editOrIssueLinks" class="feedback-edit-or-issue">
+      <EditOrIssue />
+    </div>
+  </div>
+</template>
+
+<script>
+import EditOrIssue from './EditOrIssue.vue'
+
+export default {
+  name: 'AposFeedback',
+  data: function() {
+    return {
+      voteSubmitted: false,
+      currentPath: this.$route.path
+    }
+  },
+  methods: {
+    sendFeedback: function(evtType) {
+      this.voteSubmitted = true;
+      console.log(evtType);
+
+      if (!window.ga) {
+        return;
+      }
+
+      window.ga('send', 'event', {
```

Comment:
YAAAS for using google analytics as a model/reporting layer but rolling our own dang frontend and behavior to get this The Way We Want.

Reply:
Just wait until I add the textarea input...